controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/JobMeta.java

```diff
@@ -0,0 +1,46 @@
+package com.yahoo.vespa.hosted.controller.deployment;
+
+import java.time.Instant;
+import java.util.Optional;
+
+/**
+ * Contains state information for a deployment job run by an {@link InternalBuildService}.
+ *
+ * @author jonmv
+ */
+public class JobMeta {
```

Comment:
Are you sure we need this much new stuff, it seems to me this should be the same as JobRun?

Reply:
I started out with an idea that the two should be separate. Maybe I'll merge some things as I go along. I'll consider this one carefully, as the two are quite similar.